akka-stream-tests-tck/src/test/scala/akka/stream/tck/GroupByTest.scala

```diff
@@ -18,12 +18,7 @@ class GroupByTest extends AkkaPublisherVerification[Int] {
     if (elements == 0) EmptyPublisher[Int]
     else {
       val futureGroupSource =
-        Source(iterable(elements))
-          .groupBy(1, elem => "all")
-          .prefixAndTail(0)
-          .map(_._2)
-          .concatSubstreams
-          .runWith(Sink.head)
+        Source(iterable(elements)).groupBy(1, _ => "all").prefixAndTail(0).map(_._2).concatSubstreams.runWith(Sink.head)
```

Comment:
I liked the newlines but I suppose scalafmt is not on my side today ;)

Reply:
`scalafmt` is not on my side either.